src/components/UI/Form.js

```diff
@@ -1,25 +1,52 @@
-import React from 'react';
-import PropTypes from 'prop-types';
-import Button from './Button';
+import React, { useState } from 'react';
+import { useDispatch } from 'react-redux';
+import { v4 as uuid } from 'uuid';
+import { addBook } from '../../redux/books/books';
+import classes from './Form.module.css';
 
-const Form = ({ id, labelContent }) => (
-  <form>
-    <label htmlFor={id}>
-      {labelContent}
-      <input type="text" id={id} />
-    </label>
-    <Button content="Add" />
-  </form>
-);
+const Form = () => {
+  const dispatch = useDispatch();
+  const [bookTitle, setBookTitle] = useState('');
+  const [bookCategory, setBookCategory] = useState('');
 
-Form.defaultProps = {
-  id: '',
-  labelContent: '',
-};
+  const bookTitleHandle = (e) => {
+    const title = e.target.value;
+    setBookTitle(title);
+  };
+
+  const bookCategoryHandle = (e) => {
+    const author = e.target.value;
+    setBookCategory(author);
+  };
+
+  const submitBookToStore = (e) => {
+    e.preventDefault();
+    const newBook = {
+      id: uuid(),
+      title: bookTitle,
+      category: bookCategory,
+    };
+    dispatch(addBook(newBook));
+
+    setBookTitle('');
+    setBookCategory('');
+  };
 
-Form.propTypes = {
-  id: PropTypes.string,
-  labelContent: PropTypes.string,
+  return (
+    <div className={classes.formSection}>
+      <div className={classes.line} />
+      <form onSubmit={submitBookToStore}>
+        <div className={classes.formContainer}>
+          <h2 className={classes.addNewBook}>ADD NEW BOOK</h2>
+          <div className={classes.inputsContainer}>
+            <input className={classes.inputTitle} type="text" id="title" value={bookTitle} onChange={bookTitleHandle} placeholder="Book Title" />
+            <input className={classes.inputCategory} type="text" id="category" value={bookCategory} onChange={bookCategoryHandle} placeholder="Book Category" />
+            <button className={classes.formButton} type="submit">ADD BOOK</button>
```

Comment:
- [ ] Whenever adding a book the function should generate an unique ID

Reply:
@GzimAsani 
Hey,

Thanks for your comments but I think its already show a unique id for each book !

![image](https://user-images.githubusercontent.com/88675267/141006010-8046147c-d2d4-4b5e-849c-c476f44f76a9.png)

![image](https://user-images.githubusercontent.com/88675267/141006109-712fb91d-fea6-4226-b297-0bf7e8a04c1f.png)

![image](https://user-images.githubusercontent.com/88675267/141006099-cba2001f-7467-4ee4-b4dd-b7e62b20b35b.png)